setup.py

```diff
@@ -26,7 +26,8 @@ def read_file(filename):
     package_data={},
     python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*",
     setup_requires=["setuptools_scm"],
-    install_requires=["click>=6", "six"],
+    install_requires=["click>=6", "pip", "setuptools", "six", "wheel"],
+    tests_require=["mock", "pytest"],
```

Comment:
Even if there will be test deps `python setup.py test` will run zero tests.

FIY `python setup.py test` is going to be deprecated https://github.com/pypa/setuptools/issues/1684

Reply:
`python setup.py test` is a separate issue.
This is about declaring dependencies.  Lots of tools use these declarations, other than `setuptools`.  The setuptools maintainers will find a lot of push back when they actually *deprecate* it (if deprecation=warning), and all hell will break loose when/if it is actually removed.

Reply:
> Lots of tools use these declarations, other than `setuptools`

Just out of a curiosity, which tools?

Reply:
Thanks for the reference.